Packs/EWS/Integrations/EWSO365/EWSO365.py

```diff
@@ -2277,7 +2268,7 @@ def fetch_emails_as_incidents(client: EWSClient, last_run):
 
 
 def fetch_last_emails(
-        client: EWSClient, folder_name="Inbox", since_datetime=None, exclude_ids=None
+        client: EWSClient, folder_name="Inbox", since_datetime=None, exclude_ids=None, incidentFilter='received-time'
```

Comment:
I really think a boolean would be nicer here.

Reply:
Boolean is for "this or not-this". Our choices for this parameter are "this, that, and possibly some time in the future, a number of others".
Maybe an enum, but there are no examples for me to follow.